.eslintrc

```diff
@@ -1,8 +0,0 @@
-{
-  "env": {
-    "node": true
```

Comment:
does lab set their internal eslint to node env? if not, this turns on some rules and would be worth keeping